packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -614,7 +615,15 @@ export class ConnectionManager implements IConnectionManager {
 					// We skip this delay if we're confident we're offline, because we probably just need to wait to come back online.
 					await new Promise<void>((resolve) => {
 						setTimeout(resolve, delayMs);
-						delayMs = Math.min(delayMs * 2, MaxReconnectDelayInMs);
+						// If the error has some status code, it means we probably reached to endpoint and got some error. In that case
+						// set the max time to be 8s as we want to retry at faster pace. Otherwise, wait for MaxReconnectDelayInMs at max.
+						delayMs = Math.min(
+							delayMs * 2,
+							isFluidError(origError) &&
+								origError.getTelemetryProperties().statusCode !== undefined
+								? 8000
```

Comment:
It would be great to have 8000 next to 30000 seconds as a constant, such that it's easier to see them (and ideally we have comments next to them such that anybody can better understand what they influence without need to read the rest of the logic.
I can see us eventually moving these constants to policy that maybe driver or host would have a chance to overwrite (but I'd do it right now - we do not have proper scenario to justify that work).